objects/input-object-array/script.js

```diff
@@ -1,22 +1,48 @@
 let alunos = [];
+const button = document.querySelector("button");
 
-function add(){
+function generateList() {
+  const lista = document.querySelector('.lista');
+
+  for (let i = 0; i < alunos.length; i++) {
+    const text = ("Nome: " + `${alunos[i].nome}` + " Nota: " + `${alunos[i].nota} `)
+
+    const item = document.createElement('li');
+    item.appendChild(document.createTextNode(text));
+    lista.appendChild(item);
+  }
+
+}
+
+
+function clearInput() { //Limpar input assim que submeter
+  document.querySelector('.nome').value = "";
+  document.querySelector('.nota').value = "";
+}
+
+
+function removeItems() {
+  document.querySelector('.lista').innerHTML = "";
+}
+
+function handleSubmit() {
   const nomeAluno = document.querySelector('.nome').value;
   const notaAluno = document.querySelector('.nota').value;
-  const resultado = document.querySelector('.resultado');
 
-  const dados = {
+  const notas = {
```

Comment:
```suggestion
  const aluno = {
```
Pessoalmente, chamaria esse cara de `aluno`, porque faz mais sentido para mim. Até porque o array que você tem para armazená-los se chama `alunos`.